packages/avatar-bundle/src/AvatarBundle.stylesheet.js

```diff
@@ -0,0 +1,79 @@
+import {
+  SizeMapping,
+  StyleItems,
+  stylesheet as avatarStylesheet,
+  sizes
+} from "@hig/avatar";
+
+import { spacings } from "./spacings";
+
+export const SpacingMapping = {
+  [spacings.DEFAULT]: "default",
+  [spacings.CONDENSED]: "condensed",
+  [spacings.SUPER_CONDENSED]: "superCondensed"
+};
+
+export default function stylesheet(props, themeData) {
+  const { size, spacing, borderColor, stylesheet: customStylesheet } = props;
+  const sizeString = size ? SizeMapping[size] : SizeMapping[sizes.MEDIUM_32];
+  const spacingString = spacing
+    ? SpacingMapping[spacing]
+    : SpacingMapping[spacings.DEFAULT];
+
+  let clipPath = "";
+  let boxShadow = "";
+  if (borderColor) {
+    const borderWidth = themeData[`avatarBundle.${sizeString}.border`];
+    boxShadow = `0 0 0 ${borderWidth} ${borderColor}`;
+  } else {
+    clipPath = `url(#avatar-bundle-clip-${sizeString}-${spacingString})`;
+  }
+
+  const avatarSpacingSecondItem =
+    themeData[`avatarBundle.${sizeString}.${spacingString}.spacing.secondItem`];
+  const avatarSpacingThirdItem =
+    themeData[`avatarBundle.${sizeString}.${spacingString}.spacing.thirdItem`];
+  const overflowBackground = themeData["avatarBundle.default.backgroundColor"];
+  const overflowFontColor = themeData["avatarBundle.fontColor"];
+
+  const avatarStyle = avatarStylesheet({ size }, themeData);
+  const { height: diameter } = avatarStyle[StyleItems.avatarContainer];
```

Comment:
instead of grabbing these items from the `@hig/avatar` stylesheet, isn't this info available from theme-data (height, font-family)?

Reply:
They are, but it's important that they are the exact same values as the Avatar component. This way if the Avatar values are changed for some reason, the AvatarBundle won't break. That's my thinking, at least.

Reply:
For styling the source of truth is theme-data as the `@hig/avatar` pulls from that directly, I think you just need to make sure how the sizing mapped is aligned between the two components.

As long as the component has `@hig/theme-data` as a dependency then when that gets updated `semantic-release` will also update its dependent packages with the next minor/patch version.